tests/test_phosim_repackager.py

```diff
@@ -87,28 +95,77 @@ def _check_repackaged_eimg_file(self, file_name):
         sensor = fits.open(eimg_file)[0]
 
         # Check the image is transposed
-        self.assertEqual(sensor.data.shape, (4000, 4072))
+        self.assertEqual(sensor.data.shape, (4004, 4096))
 
         # Check the header imformation
         header = sensor.header
-        self.assertEqual(header['RAFTNAME'], 'R22')
-        self.assertEqual(header['SENSNAME'], 'S00')
-        self.assertEqual(header['ROTANGLE'], 0.0)
+        self.assertEqual(header["RAFTBAY"], "R22")
+        self.assertEqual(header["CCDSLOT"], "S22")
+        self.assertEqual(header["RA"], 0.0)
 
-    def _get_num_of_file_in_dir(self, directory):
+    def test_process_visit_amp_image(self):
+
+        num_file = self._get_num_of_file_in_dir(self.tmp_test_dir)
+        self.assertEqual(num_file, 0)
 
-        return len([name for name in os.listdir(directory)
-                   if os.path.isfile(os.path.join(directory, name))])
+        self.phoSim_repackager.process_visit(
+            self.test_data_dir_amp, out_dir=self.tmp_test_dir, prefix="lsst"
+        )
 
-    def test_mef_filename_eimage(self):
+        num_file = self._get_num_of_file_in_dir(self.tmp_test_dir)
+        self.assertEqual(num_file, 1)
 
-        out_dir = os.path.join(os.sep, "path", "of", "output")
+    def test_repackage_amp_image(self):
 
-        outfile = PhoSimRepackager.mef_filename_eimage(
-            self.eimg_file_path, out_dir=out_dir)
+        num_file = self._get_num_of_file_in_dir(self.tmp_test_dir)
+        self.assertEqual(num_file, 0)
+
+        self.phoSim_repackager.repackage(
+            self.amp_file_paths, out_dir=self.tmp_test_dir
+        )
+
+        num_file = self._get_num_of_file_in_dir(self.tmp_test_dir)
+        self.assertEqual(num_file, 1)
+
+        file_name = self.repackaged_amp_file_name
+        self._check_repackaged_amp_file(file_name)
+
+    def _check_repackaged_amp_file(self, file_name):
+
+        amp_file = os.path.join(self.tmp_test_dir, file_name)
+        print(self.tmp_test_dir, file_name)
+        hdul = fits.open(amp_file)
```

Comment:
Did you close the file after the opening?

Reply:
Added closing file.